taccsite_cms/templates/djangocms_forms/form_template/default.html

```diff
@@ -0,0 +1,82 @@
+{# https://github.com/avryhof/djangocms-forms/blob/30f4643/djangocms_forms/templates/djangocms_forms/form_template/default.html #}
```

Comment:
The changes to this copied template are commented; the comments begin with `{# TACC: `. The changes are:
1. Load stylesheet ([which could be removed…](…)).
2. Remove redundant script load (fixes negligible console error) (could be done as PR to plugin repo).

**Should I forgo these changes i.e. delete this file?**

Reply:
Removed in 8a46a21's update of #498's templates.

1. CSS was moved to `django.cms.forms.css`.
2. PR was created to fix the redundancy in the plugin repo:
  https://github.com/avryhof/djangocms-forms/pull/12